op-service/txmgr/queue_test.go

```diff
@@ -158,7 +158,7 @@ func TestQueue_Send(t *testing.T) {
 				{},
 			},
 			nonces: []uint64{0, 1},
-			total:  3 * time.Second,
+			total:  1 * time.Second,
```

Comment:
now that the abort due to low nonce happens immediately this timing had to be adjusted.

Reply:
nice, signs of progress